src/main/resources/static/views/interest/InvestorContact.html

```diff
@@ -0,0 +1,32 @@
+<div class="container" ng-init="getContact()">
+    <div class="row">
+        <div class="col-xs-12 col-sm-10 col-md-8 col-lg-8 col-lg-offset-2" style="margin-top: 50px;">
+            <div class="panel-card panel-success-card">
+                <div class="text-center"><h3>INVESTMENT INTEREST</h3></div>
+                <div class="panel-body">
+                    <div class="row">
+                        <div class="col-xs-12 text-center">
+                            <h4><b>Our goal:</b></h4>
+                            <p>{{investorContact.investmentGoal}}</p>
+                            <p>{{investorContact.description}}</p>
+                            <h4 class="text-left"><b> Contacts:</b></h4>
+                            <h5 style="font-size: 15px" class="text-left"><i class="fa fa-user fa-lg" aria-hidden="true"></i> Name: {{investorContact.investor.name}}  </h5>
+                            <h5 style="font-size: 15px" class="text-left"><i class="fa fa-envelope fa-lg" aria-hidden="true"></i> Email: {{investorContact.investor.email}}  </h5>
+                            <h5 style="font-size: 15px" class="text-left"><i class="fa fa-phone-square fa-lg" aria-hidden="true"></i> Phone: {{investorContact.investor.contact.phoneNumbe}}  </h5>

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this <i> tag by <em>. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)